Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM python:3.5
+MAINTAINER Onion Tech <webtech@theonion.com>
+
+RUN apt-get update \
+    && apt-get upgrade -y \
+    && apt-get update \
+    && apt-get install -y \
+        git-core \
+        libmemcached-dev \
+        libpq-dev \
+        postgresql-client-9.4 \
+        vim \
+    && rm -rf /var/lib/apt/lists/*
+
+# Deploy-specific python packages
+# - Django runs inside uWSGI
+# - docker-py required for Docker-assigned ephemeral port detection from within container
+RUN pip install "uwsgi>=2.0.11.1,<=2.1" \
+                "docker-py==1.4.0"
```

Comment:
Don't need either of these python libs (unless you are planning to run "example" app via UWSGI -- then you'd just need 'uwsgi')